lighthouse-core/report/html/templates.html

```diff
@@ -669,24 +669,24 @@
         <!-- Just fast and reliable. -->
         <g class="lh-gauge--pwa__component lh-gauge--pwa__fast-reliable-badge" transform="translate(20, 29)">
           <path fill="url(#lh-gauge--pwa__fast-reliable__shadow-gradient)" d="M33.63 19.49A30 30 0 0 1 16.2 30.36L3 17.14 17.14 3l16.49 16.49z"/>
-          <use xlink:href="#lh-gauge--pwa__fast-reliable-badge" />
+          <use href="#lh-gauge--pwa__fast-reliable-badge" />
```

Comment:
apparently `xlink:href` is [old and busted](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href)